core/core-funcs.el

```diff
@@ -119,6 +119,14 @@ Supported properties:
   (re-search-forward anchor-text)
   (beginning-of-line)
   (show-subtree)
+  (setq-local org-emphasis-alist '(("*" bold)
+                                   ("/" italic)
+                                   ("_" underline)
+                                   ("=" org-verbatim verbatim)
+                                   ("~" org-kbd)
+                                   ("+"
+                                    (:strike-through t))))
+
```

Comment:
Awesome, but can this be used for all README.org files as well though? :O
so as to have a guidance while editing READMEs from Layers

On Tue, Jun 9, 2015 at 5:23 AM, Tu Do <notifications@github.com> wrote:

> In core/core-funcs.el
> <https://github.com/syl20bnr/spacemacs/pull/1906#discussion_r31999605>:
>
> > @@ -119,6 +119,14 @@ Supported properties:
> >    (re-search-forward anchor-text)
> >    (beginning-of-line)
> >    (show-subtree)
> > +  (setq-local org-emphasis-alist '(("*" bold)
> > +                                   ("/" italic)
> > +                                   ("_" underline)
> > +                                   ("=" org-verbatim verbatim)
> > +                                   ("~" org-kbd)
> > +                                   ("+"
> > +                                    (:strike-through t))))
> > +
>
> It won't break your org-emphasis-alist for this reason:
>
> setq-local only sets the variable value for that buffer. setq sets that
> variable globally.
>
> So, only buffer opened with spacemacs/open-file is affected by the above
> setting to org-emphasis-alist.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/1906/files#r31999605>.
>